.changeset/fair-buttons-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-common': minor
+---
+
+Adds a new export for the generated client, `@backstage/plugin-search-common/client`.
```

Comment:
Needs an update now right? same with the other changesets

Reply:
Updated!